jvm/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -66,9 +68,11 @@ case class TransformPreOverrides() extends Rule[SparkPlan] {
         new BatchScanExecTransformer(plan.output, plan.scan, newPartitionFilters)
       case plan: FileSourceScanExec =>
         logDebug(s"Columnar Processing for ${plan.getClass} is currently supported.")
+        val realOutput = BackendsApiManager.getTransformerApiInstance.
```

Comment:
what about BatchScanExecTransformer?

Reply:
How to trigger the BatchScanExecTransformer

Reply:
Please refer to the test case: GlutenClickHouseDSV2Suite